docs/subquery_network/kepler/indexers/dictionary-restore.md

```diff
@@ -0,0 +1,102 @@
+| Moonbeam  | `QmRwisx41SRrR8fhNTRpKetNdpP7SaNkRAVrRgcdwoEtCH` | TBC           |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+
+You can download the snapshot either from the s3 bucket URL or the BitTorrent magnet link:
+
+### Download Snapshot
+
+Downloading via Bittorrent will be faster, you can use your favourite bittorent client or install Aria2
+
+```bash
+sudo add-apt-repository universe
```

Comment:
is aria2 only available in universe, I don't remember I use that repo when installing aria2.
@Jianzhuo

Reply:
Yes， For Ubuntu, and Linux Mint systems use `sudo apt-get install aria2` to install aria2.